src/packages/auth/adapters/NoAuthAdapter.ts

```diff
@@ -0,0 +1,20 @@
+import { getEnv } from '../../../env.js';
+import { generateJwt, getUserRoles } from '../functions.js';
+import type { AuthAdapter, ValidateResponse } from '../types.js';
+
+export const NoAuthAdapter: AuthAdapter = {
+  logout: async (): Promise<boolean> => true,
+  validate: async (): Promise<ValidateResponse> => {
+    const { DEFAULT_ROLE_NO_AUTH, ALLOWED_ROLES_NO_AUTH } = getEnv();
+
+    console.log('auth disabled, returning default roles');
+    const userId = 'default_user';
+    const { allowed_roles, default_role } = await getUserRoles(userId, DEFAULT_ROLE_NO_AUTH, ALLOWED_ROLES_NO_AUTH);
```

Comment:
This implementation feels like it will cause issues on deployments with multiple users but not using auth, since it will regenerate everyone's token into this `default_user`. The logic in `FakeAuthAdapter` seems like it should be here.

Reply:
`NoAuthAdapter` would be used in cases where user distinction isn't desired, i.e. running tests in CI, developing on a local deployment, etc